include/soci/session.h

```diff
@@ -150,6 +150,19 @@ class SOCI_DECL session
     std::string empty_blob();
     std::string nvl();
 
+    // And some functions to help with writing portable DML statements.
+
+    // Get the name of the dummy table that needs to be used in the FROM clause
+    // of a SELECT statement not operating on any tables, e.g. "dual" for
+    // Oracle. The returned string is empty if no such table is needed.
+    std::string get_dummy_from_table() const;
```

Comment:
I expect this to be rarely used, i.e. I think most people would just use `get_dummy_from_clause()` anyhow (and I even didn't want to make this one public at all initially, although I finally ended up by doing it because why not, basically...), so I decided that keeping the name similar to the other function was more important than making it shorter.